services/web/client/source/resource/form/service.json

```diff
@@ -0,0 +1,186 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Add a new service",
+    "description": "Please provide with all the necessary information about your service.",
+    "type": "object",
+    "properties": {
+        "contributor": {
+            "type": "object",
+            "title": "Contributor information",
+            "required": [
+                "firstName",
+                "lastName",
+                "email",
+                "affiliation"
+            ],
+            "properties": {
+                "firstName": {
+                    "type": "string",
+                    "title": "First name"
+                },
+                "lastName": {
+                    "type": "string",
+                    "title": "Last name"
+                },
+                "email": {
+                    "type": "string",
+                    "title": "Email"
+                },
+                "affiliation": {
+                    "type": "string",
+                    "title": "Affiliation"
+                }
+            }
+        },
+        "serviceInfo": {
+            "type": "object",
+            "title": "Service information",
+            "required": [
+                "description", 
+                "commandLine"
+            ],
+            "properties": {
+                "description": {
+                    "type": "string",
+                    "title": "Description of the service",
+                    "description": "Please provide with detailed information about how to run the service and any dependencies required."
+                },
+                "commandLine": {
+                    "type": "string",
+                    "title": "Command line interface to run service",
+                    "description": "In the command line, what command is used to run the code? E.g. In Matlab Command Window: 'myproject(input1, input2, input3)'"
```

Comment:
suggestion: "Describe how the service is run."
Can't you set examples in jsonschema? you might even be able to put several ones.
examples:
- Matlab: myproject(input1, input2, input3)
- console: myproject input1 input2 input3

Reply:
What you set inside the description will be use as helper text for the user. You mean like examples of correct data for that field? Not like in the openapi specs. If you want to display it to the user, then there is the description. If you want to actually validate the input, then you have to use patterns. More complex validation, like different patterns depending on other fields data, are not implemented yet... but they are all feasible to implement in next iterations

Reply:
:+1: